### PR TITLE
replace custom flake8 test with ament_pep8 / ament_pyflakes invocation

### PR DESCRIPTION
**Pull request by dirk-thomas**
Connects to ament/ament_tools#22.

### CODE REVIEW

**Comment by esteve** [comment]
+1

**Comment by dirk-thomas** [comment]
(Before merging I have to replace the temporary branch `split_linter_packages_in_python_and_cmake` in the travis file.)

File: .travis.yml [hunk 1]
```diff
@@ -3,19 +3,33 @@ python:
...
+  - python setup.py install
+  - cd ../..
+  - cd ament_lint/ament_pyflakes
+  - python setup.py install
+  - cd ../..
 script:
   - python setup.py nosetests -s
+  - ament_copyright ament_tools scripts test
+  - ament_pep8 ament_tools scripts test
+  - ament_pyflakes ament_tools scripts test
```
Comment (wjwwood) [inline .travis.yml]: I would prefer these to be part of the nose tests, as the flake8 test was before. Why did you do it this way instead?

**Comment by wjwwood** [comment]
+1

**Pull request merged**